src/ESM.t.sol

```diff
@@ -333,15 +333,32 @@ contract ESMTest is DSTest {
         assertEq(end.live(), 0);
     }
 
-    function testFail_file_revoked_gov() public {
+    function testFail_file_after_denied() public {
         esm = new ESM(address(gem), address(end), pauseProxy, 10_000 * WAD);
         esm.deny(address(this));
         assertEq(esm.min(), 10_000 * WAD);
 
         esm.file("min", 20_000 * WAD);
     }
 
-    function testFail_file_not_gov() public {
+    function test_file_relied_address() public {
+        // This is the pattern we will follow for the deployer:
+        // 1. deploy the contract
+        // 2. rely on the governance pause proxy
+        // 3. deny the deployer to revoke access
+        // Now the pause proxy should be able to file new mins
```

Comment:
Note: the pause proxy we are referring here is actually the `usr` which in the real deployment should be the same than `pauseProxy`.

Reply:
let me know if those changes make it more clear

Reply:
Yes looks good.